pbr/pbr.py

```diff
@@ -119,6 +119,9 @@ def main():
         ball_data = random.choice(balls)
         grass_data = random.choice(grasses)
 
+        if frame_num == 1:
+            field.update(grass_data, config["field"])
+
```

Comment:
how come this was added

Reply:
had to generate the field first to get the field dimensions but im gonna change this so that I can just grab some values from the config files somehwere